index.bs

```diff
@@ -3763,41 +3775,54 @@ nothrow>WritableStreamDefaultControllerError ( <var>controller</var>, <var>error
-  faster than it can be transformed and consumed.
+  and a <a>writable stream</a>. <a>Chunks</a> that travel from the <a>readable stream</a> to the <a>writable stream</a>
+  will be transformed as they pass through the transform stream. <a>Backpressure</a> is respected, so data will not be
+  read faster than it can be transformed and consumed.
 
   <pre><code class="lang-javascript">
-    readableStream.pipeThrough(transformStream)
-                  .pipeTo(writableStream)
+    readableStream
+      .pipeThrough(transformStream)
+      .pipeTo(writableStream)
       .then(() => console.log("All data successfully transformed!"))
       .catch(e => console.error("Something went wrong!", e));
   </code></pre>
 </div>
 
-<div class="example" id="example-transform-stream-accessors">
-  You can also use the {{TransformStream/readable}} and {{TransformStream/writable}} accessors of a transform stream
-  directly with all the usual interfaces of a <a>readable stream</a> and <a>writable stream</a>. In this example we
+<div class="example" id="example-transform-stream-properties">
+  You can also use the {{TransformStream/readable}} and {{TransformStream/writable}} properties of a transform stream
+  directly to access the usual interfaces of a <a>readable stream</a> and <a>writable stream</a>. In this example we
   supply input data to the stream using the <a>writer</a> interface. The output is then piped to
   <code>anotherWritableStream</code>.
 
   <pre><code class="lang-javascript">
     const writer = transformStream.writable.getWriter();
-    writer.write('input chunk');
+    writer.write("input chunk");
     transformStream.readable.pipeTo(anotherWritableStream);
   </code></pre>
 </div>
 
 <div class="example" id="example-transform-identity">
-  An identity transform is a type of transform stream which forwards all input chunks unchanged to the output. This is
-  the default behaviour of {{TransformStream}} when no <code>transform</code> method is supplied to the
-  constructor. One use of identity transforms is to add additional buffering to a pipe. In this example we add extra
-  buffering between <code>readableStream</code> and <code>writableStream</code>.
+  One use of <a>identity transform streams</a> is to easily convert between readable and writable streams. For example,
+  the {{fetch()}} API accepts a readable stream <a for="request" lt="body">request body</a>, but it can be more
+  convenient to write data for uploading via a writable stream interface. Using an identity transform stream addresses
+  this:
+
+  <pre><code class="lang-javascript">
+    const { writable, readable } = new TransformStream();
+    fetch("...", { body: readable }).then(response => /* ... */);
+
+    writable.write(new Uint8Array([0x73, 0x74, 0x72, 0x65, 0x61, 0x6D, 0x73, 0x21]));
+    writable.close();
+  </code></pre>
+
+  Another use of identity transform streams is to add additional buffering to a <a>pipe</a>. In this example we add
```

Comment:
Is it possible to have a separate id for the buffering example without disrupting the readability of this section too much?

Reply:
Sure, we can just make it another `<div>`, not a big deal I think.

Reply:
+1